dll/win32/shell32/lang/en-US.rc

```diff
@@ -873,6 +873,7 @@ BEGIN
     IDS_CANTSHOWPROPERTIES "Unable to show properties (Error Code: %lu)."
     IDS_CANTDISCONNECT "Unable to disconnect (Error Code: %lu)."
     IDS_NONE "(None)"
+    IDS_CMD_VERB "Command Prompt here"
```

Comment:
Please move all the instances of IDS_CMD_VERB (in all languages) to where the other _VERB strings are defined.

Reply:
Moved.